src/Symfony/Bundle/FrameworkBundle/Command/WorkflowDumpCommand.php

```diff
@@ -109,4 +111,33 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
         return 0;
     }
+
+    public function complete(CompletionInput $input, CompletionSuggestions $suggestions): void
+    {
+        if ($input->mustSuggestArgumentValuesFor('name')) {
+            $suggestions->suggestValues($this->getAvailableNameArguments());
+        }
+
+        if ($input->mustSuggestOptionValuesFor('dump-format')) {
+            $suggestions->suggestValues($this->getAvailableDumpFormatOptions());
+        }
+    }
+
+    private function getAvailableNameArguments(): array
+    {
+        $names = [];
```

Comment:
why not inject them from the `FrameworkExtension` like inside the `workflow.registry`?

Reply:
I see container get definition but cannot inject into command 😢

Reply:
Why?

Reply:
I think just do not know how. If good understand I need inection configuration to load this one 

```
$config['workflows'] 
```

Reply:
> I think just do not know how. If good understand I need inject configuration to load this one

pass this (or a new array computed from this list) inside the command construct, then in the execute and the complete methods you can get those and handle the logic

Reply:
@StaffNowa as discussed on slack, I've sent you an example you can pick then continue working on this :)